### PR TITLE
German Translation of User Guides (General 1/2)

### PR DESCRIPTION
**Pull request by Wobole**
German translation of the following user guides:

- How to make an offline backup
- How to run a node on VPS
- Importing the Monero blockchain
- Monero Tools

### CODE REVIEW

**Comment by erciccione** [comment]
Hey @stmax82, would you be interested in reviewing this and the other open pull requests adding German translations? We'll give you a small reward in XMR as a thank you :)

**Comment by stmax82** [comment]
> Hey @stmax82, would you be interested in reviewing this and the other open pull requests adding German translations? We'll give you a small reward in XMR as a thank you :)

@erciccione is it the 5 PRs about User Guides General 1+2, Wallets 1+2, Mining and Recovery? I'll start working on it, could take a bit though because other jobs are keeping me quite busy right now.

File: _i18n/de/resources/user-guides/importing_blockchain.md [hunk 1]
```diff
@@ -1,62 +1,62 @@
...
+Verwende **niemals** die gefährliche, ungeprüfte Importoption; diese ist ausschließlich für Experten gedacht. Insbesondere solltest du sie nicht bei einer aus dem Internet, einschließlich von der offiziellen Webseite heruntergeladen Blockchain verwenden. Nur wenn du a) eine Datei importierst, die du eigenhändig lokal exportiert hast *und* b) du absolut sicher bist, dass die Datei vor dem Exportieren gründlich und vollständig überprüft wurde, ist ihr Gebrauch wirklich sicher.
 
-### Step 1
+### Schritt 1
 
-Download the Current bootstrap from https://downloads.getmonero.org/blockchain.raw; you can skip this step if you are importing the blockchain from another source.
+Lade die aktuelle Bootstrap von https://downloads.getmonero.org/blockchain.raw herunter. Wenn du die Blockchain aus einer anderen Quelle importierst, kannst du diesen Schritt überspringen.
```
Comment (stmax82) [inline _i18n/de/resources/user-guides/importing_blockchain.md:13]: Not sure how to translate "Bootstrap", would try to pick another word for it..

File: _i18n/de/resources/user-guides/vps_run_node.md [hunk 2]
```diff
@@ -1,48 +1,48 @@
...
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
 
-# monerod
+# Monerod
 
-`monerod` is the daemon software that ships with the Monero tree. It is a console program, and manages the blockchain. While a bitcoin wallet manages both an account and the blockchain, Monero separates these: `monerod` handles the blockchain, and `monero-wallet-cli` handles the account.
+`monerod` ist die Hintergrunddienst-Software, die mit der Monero-Basissoftware einhergeht. Es ist ein Konsolenprogramm, das die Blockchain managt. Während ein Bitcoin-Wallet gleichzeutig das Konto wie auch die Blockchain managt, ist dies bei Monero separiert: `monerod` bewerkstelligt die Blockchain und `monero-wallet-cli` das Konto.
 
-This guide assumes you have already set up your VPS account and are using SSH to tunnel into the server console.
+Diese Anleitung setzt voraus, dass du bereits einen VPS-Account angelegt hast und via SSH-Tunnel zum Konsolenserver durchdringst.
```
Comment (stmax82) [inline _i18n/de/resources/user-guides/vps_run_node.md:7]:
```suggestion
Diese Anleitung setzt voraus, dass du bereits einen VPS-Account angelegt hast und dich via SSH-Tunnel zur Server-Konsole verbinden kannst.
```

File: _i18n/de/resources/user-guides/vps_run_node.md [hunk 3]
```diff
@@ -1,48 +1,48 @@
...
+`monerod` ist die Hintergrunddienst-Software, die mit der Monero-Basissoftware einhergeht. Es ist ein Konsolenprogramm, das die Blockchain managt. Während ein Bitcoin-Wallet gleichzeutig das Konto wie auch die Blockchain managt, ist dies bei Monero separiert: `monerod` bewerkstelligt die Blockchain und `monero-wallet-cli` das Konto.
 
-This guide assumes you have already set up your VPS account and are using SSH to tunnel into the server console.
+Diese Anleitung setzt voraus, dass du bereits einen VPS-Account angelegt hast und via SSH-Tunnel zum Konsolenserver durchdringst. 
 
 ## Linux, 64-bit (Ubuntu 16.04 LTS)
 
-### Make sure that port 18080 is open
-`monerod` uses this port to communicate with other nodes on the Monero network.
+### Stelle sicher, dass Anschluss 18080 geöffnet ist
```
Comment (stmax82) [inline _i18n/de/resources/user-guides/vps_run_node.md:11]:
```suggestion
### Stelle sicher, dass Port 18080 geöffnet ist
```

Port is fine in German when talking about network ports.

**Review by stmax82 — CHANGES_REQUESTED** [review]

**Comment by erciccione** [comment]
> @erciccione is it the 5 PRs about User Guides General 1+2, Wallets 1+2, Mining and Recovery?

Yes. Basically all PRsopened by @wobole.

Thanks :)

File: _i18n/de/resources/user-guides/importing_blockchain.md (hunk 1 above)
Comment (erciccione) [reply _i18n/de/resources/user-guides/importing_blockchain.md:13]: Yeah that word is problematic for translators. I hope we will use it less in the future.

**Review by erciccione — APPROVED** [review]

**Comment by stmax82** [comment]
> > @erciccione is it the 5 PRs about User Guides General 1+2, Wallets 1+2, Mining and Recovery?
> 
> Yes. Basically all PRsopened by @Wobole.
> 
> Thanks :)

@erciccione @Wobole I think I'm done with all 5 PRs. Hope I didn't miss any parts, otherwise let me know! Hats off to the work you put into this (especially the Multisig part must've been tough to write and translate :)

**Comment by erciccione** [comment]
@stmax82 is there a way to contact you privately? I'm on Freenode (IRC) and matrix with the same nickname.

**Comment by stmax82** [comment]
@erciccione I sent you a message from @stmax:matrix.org

**Comment by Wobole** [comment]
> @erciccione @Wobole I think I'm done with all 5 PRs. Hope I didn't miss any parts, otherwise let me know! Hats off to the work you put into this (especially the Multisig part must've been tough to write and translate :)

Thank you so much for your review, @stmax82! And also a big thanks for teaching me that it's "sich verbinden" - I anglicized a bit there, never to be forgotten again. :)

**Pull request merged**